__tests__/app.test.js

```diff
@@ -0,0 +1,57 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data")
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+
+//closes all databases after the test suite has run.
+afterAll(() => {
+    if (db.end) db.end();
```

Comment:
We don't need the if statement here. It's fine just to return the `db.end()`

Reply:
ok cool I'll simplify it.